vars.py

```diff
@@ -95,6 +95,7 @@ def set_start_vars(self):
         self.ne_battles = config_vars.get("ne_battles", 0)  # Default to 0
         # Decides in which zone we charge Rikku OD after reaching Zanarkand.
         self.nea_zone = config_vars.get("nea_zone", 0)
+        self.generate_saves = config_vars.get("generate_saves", 0)
```

Comment:
We need to update `config.yaml.example` to include this config key/value.

Reply:
I believe I imported that to the branch, it should be part of the PR. If not... let me know and I'll add it to the example file in main.

Reply:
I don't see `config.yaml.example` as a file with changes.